packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -223,6 +228,7 @@ export const Toast = (props: ToastProps) => {
             type={type}
             hasCloseButton={hasCloseButton}
             onClose={close}
+            contentProps={contentProps}
```

Comment:
currently, the content (message) can take a custom classname, but the toast itself cannot.

suggestion: replace `contentProps` with `domProps` which is an object with two fields: `content` and `toast`
```js
toaster.positive('hello', {
  domProps: {
    toast: { className: 'my-toast' },
    content: { className: 'my-toast-content' },
  }
});
```

Reply:
better?

Reply:
done

Reply:
I believe my comment (https://github.com/iTwin/iTwinUI/pull/1510#discussion_r1305591750) is related to this thread.